server/controllers/parties.js

```diff
@@ -0,0 +1,121 @@
+import partiesDb from '../models/parties';
+
+// Create Party
+class parties {
+  static create(req, res) {
+    if (!req.body.name) {
+      return res.status(400).json({
+        status: 400,
+        error: 'Party Name is required',
+      });
+    }
+    if (!req.body.hqAddress) {
+      return res.status(400).json({
+        status: 400,
+        error: 'Address is required',
+      });
+    }
+    if (!req.body.logoUrl) {
+      return res.status(400).json({
+        status: 400,
+        error: 'logoUrl is required',
+      });
+    }
+
+    const alreadyExists = partiesDb
+      .find(party => party.name.toLowerCase() === req.body.name.toLowerCase());
+
+    if (alreadyExists) {
+      return res.status(400).json({
+        status: 400,
+        error: 'party already exists',
+      });
+    }
+
+    let createdParty;
+
+    if (!alreadyExists) {
```

Comment:
There is no need to check if the party does not exist here because if it exist it wont get to this point
Kindly remove the code from the if { } block

Reply:
@ibkadeeko

Reply:
ok
Sorry had to go do something quickly